_includes/sidebar.html

```diff
@@ -84,7 +84,7 @@
         <span class="icon-border order-2"></span>
       {% endif %}
 
-      <span id="mode-toggle-wrapper" class="order-1">
+      <span id="mode-toggle-wrapper" class="order-1" tabindex="0">
```

Comment:
😂 Sorry again, I missed the `tabindex` of other icons, maybe you need to make other changes:

1. Line 74 of [`_includes/sidebar.html`](https://github.com/bhavykhatri/jekyll-theme-chirpy/blob/e73e3b26801830b64bb16c6547a58afdf23f76b1/_includes/sidebar.html#L74) should be changed as follow:

```diff
- {% unless site.theme_mode %}class="order-{{ forloop.index | plus: 2 }}"{% endunless %}
+ {% unless site.theme_mode %}class="order-{{ forloop.index | plus: 2 }}" tabindex="{{ forloop.index | plus: 2 }}"{% endunless %}
```

2.  Maybe `#mode-toggle-wrapper`  would be better to use `tabindex="1"`, so we can get index from `1` to `n`

```diff
- <span id="mode-toggle-wrapper" class="order-1" tabindex="0">
+ <span id="mode-toggle-wrapper" class="order-1" tabindex="1">
```

Reply:
For both of the points, anchor tag are already tab focusable therefore no need for the changes. Tabindex are only used in special elements where the element like span is not tab focusable but to make it tabfocusable.

Reply:
https://jsfiddle.net/rq7j1ea8/

Checkout this js fiddle to understand more

have a look at this documentation if you have time https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

Reply:
Thank you for your reminder, I have actually read this article(<https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex>) before

I tested your PR code. If we don’t add `tabindex` to other icons, we will get the weird focus order as mentioned in the attached image above:

![tabindex](https://user-images.githubusercontent.com/26280748/144051564-ff1e9cbb-ba54-445a-b2e4-407bfa9e8846.png)

Reply:
NVM, it may be the best to automatically determine the tab order of other elements, so this PR is merged. Thank you for your time！